### ISSUE TITLE
Dashboard / Profile Display & Filtering Options

### ISSUE THREAD

**Issue by UFU608** [theCrag/website#2213]
I have a few suggestions for the dashboard / profile pages that I sum up here in 2 separate "Issues". This one is related to the filtering options under the "Your Profile" menue. 

The list of ascents there and the pyramid and other graphics should be filterable by date (start date / end date) and many more filtering options like country, crag and essentially any other database field (like route length, climbed with, etc....).

**Comment by scd** [comment]
Yup. We also need to add more graph types and base the graphs on the filters. In other words we want to make it more generic. We should be able to do graphs of you and your friend or you compared to your friend.

**Comment by UFU608** [comment]
Yes!! By when can this be done ;-)??


> On 29 May 2016, at 16:17, Simon Dale <notifications@github.com> wrote:
> 
> Yup. We also need to add more graph types and base the graphs on the filters. In other words we want to make it more generic. We should be able to do graphs of you and your friend or you compared to your friend.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/theCrag/website/issues/2213#issuecomment-222346616>, or mute the thread <https://github.com/notifications/unsubscribe/ASRnSK1o6ME17Z6zce2imjRDOrIxeOx4ks5qGT2CgaJpZM4IpOWR>.
> 

---------------------------------------
rouletout608@gmail.com
www.rouletout.ch
---------------------------------------

**Comment by scd** [comment]
There is currently no timeline for this. I will not be able to assess a time line on this until after we have released the mobile app.

**Comment by Mdemaillard** [comment]
related to #1984
